metacat-common-server/src/main/java/com/netflix/metacat/common/server/monitoring/LogConstants.java

```diff
@@ -13,95 +13,109 @@
 
 package com.netflix.metacat.common.server.monitoring;
 
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
+//CHECKSTYLE:OFF
 /**
  * Log constants.
  */
 public enum LogConstants {
     /**
      * General logging constants.
      */
-    GlobalPrefix("dse"), AppPrefix(GlobalPrefix + ".metacat"),
+    AppPrefix("metacat"),
+
+    /**
+     * evnets.
+     */
+    CounterEventAsync(AppPrefix + ".events.count.Async"),
+    CounterEventSync(AppPrefix + ".events.count.Sync"),
+
+    /**
+     * thrift request.
+     */
+    CounterThrift(AppPrefix + ".thrift.count.request"),
+
+    /**
+     * hive sql lock error.
+     */
+    CounterHiveSqlLockError(AppPrefix + ".count.hiveSqlLockError"),
+    CounterHiveExperimentGetTablePartitionsFailure(AppPrefix + ".hive.count.experimentGetPartitionsFailure"),
+
+    /**
+     * metacat request.
+     */
+    CounterRequestCount(AppPrefix + ".count.request"),
+
+
+    /**
+     * Notifications.
+     */
+    CounterSNSNotificationPartitionAdd(AppPrefix + ".notifications.count.partitionsAdd"),
```

Comment:
This is for the consistency of naming, i.e. the component.spectatorType.measureName. The type can be count, gauge, timer.